assets/goal-bot/README.md

```diff
@@ -0,0 +1,15 @@
+# Goal Bot
+
+## Introduction
+
+This folder holds documentation and design related to "goal-bot".
+
+This is a very early version of InferGPT that is being used to gradually expand the features supported. When reviewing the intended long-term design, it is recommended to review the top-level [README](../README.md).
+
+## Architecture
+
+High Level Architecture of the initial version of "goal-bot"
+
+> This is work-in-progress and will actively change.
```

Comment:
Cool GitHub markdown thing if interested, you can make a Note section here that pops out on the rendered few like this 

```suggestion
> [!NOTE]
> This is work-in-progress and will actively change.
```

See more info [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)